### DISCUSSION TITLE
Feature Request: Synchronized Album Management with Reimport and Duplicate Avoidance

### DISCUSSION THREAD

**Discussion by TKaluza** [immich-app/immich#5614] [General]
**Introduction**
Following the proposed feature for renaming [exported images with timestamps and album names](https://github.com/immich-app/immich/discussions/5612), this feature request aims to extend the functionality of the Immich app to manage shared albums more effectively. The primary focus is to allow users to reimport shared albums with updates (additions or deletions of images) while avoiding duplicates and efficiently linking to existing images.

**Proposed Feature**
1. **Reimport of Updated Shared Albums**: Users should be able to reimport an album that has been previously exported, shared, and modified (e.g., by friends or family members adding or deleting photos - yes there are person that do not want to use this great app).
2. **Duplicate Avoidance**: The app should recognize images previously imported and avoid creating duplicates. This can be achieved by checking the EXIF data or the unique naming format proposed in the related feature request.
3. **New and Existing Image Management**: Upon reimport, the app should create a new album which:
    - Adds newly found images that were not part of the original export.
    - Links to existing images already present in the Immich app, rather than importing them again.

**Conclusion**
This feature would enhance the collaborative experience of sharing and managing photo albums in the Immich app. It provides a way to update shared albums while maintaining organization and avoiding redundant storage usage. This would be particularly useful for users who frequently share albums with a group that for differen reasons does not have acces or want to work with immich.

**Comment by aisbergde** [comment]
deduplication is a general feature request, when importing assets, moving them and so on. But it should happen not based on metadata, but based in content (a hash key). I don't know, what is the best solution, if the asset is the same, but metadata is different.
